packages/contracts-bedrock/test/L2OutputOracle.t.sol

```diff
@@ -393,6 +417,16 @@ contract L2OutputOracle_deleteOutputs_Test is L2OutputOracle_Initializer {
         vm.expectRevert("L2OutputOracle: cannot delete outputs that have already been finalized");
         oracle.deleteL2Outputs(latestOutputIndex);
     }
+
+    /// @dev Tests that `deleteL2Outputs` reverts for finalized outputs.
+    function test_deleteL2Outputs_unAuthed_reverts() external {
+        proposeAnotherOutput();
+        uint256 latestOutputIndex = oracle.latestOutputIndex();
+
```

Comment:
This could be made stronger with as `assertFalse` on `isProposalManager`

Reply:
Will do